modular_zubbers/master_files/code/datums/id_trim/jobs.dm

```diff
@@ -0,0 +1,46 @@
+//New honorifics
```

Comment:
we don't use master_files, put this in `modular_zubbers/code/datums/id_trim/jobs.dm`